src/app/data-model/Nullable.h

```diff
@@ -48,6 +48,7 @@ struct Nullable : protected Optional<T>
     // Pull in APIs that make sense on Nullable with the same names as on
     // Optional.
     using Optional<T>::Value;
+    using Optional<T>::ValueOr;
```

Comment:
This does not seem to be used in this PR... but ok.

Reply:
Fixed